### PR TITLE
Virtual table to list thread pool information.

### PR DESCRIPTION
**Pull request by nirbhayc**

### CODE REVIEW

File: sqlite/ext/comdb2/threadpools.c [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+
+    case COLUMN_MAX_QUEUE_OVERRIDE:
+        sqlite3_result_int(ctx, thdpool_get_maxqueueoverride(pool));
+        break;
+    case COLUMN_MAX_QUEUE_AGE_MS:
+        sqlite3_result_int(ctx, thdpool_get_maxqueueagems(pool));
+        break;
+    case COLUMN_EXIT_ON_CREATE_FAIL:
+        sqlite3_result_text(
+            ctx, YESNO(thdpool_get_exit_on_create_fail(pool) == 1), -1, NULL);
```
Comment (riverszhang89) [inline sqlite/ext/comdb2/threadpools.c]: Zero for false and non-zero for true. So you don't need to compare the value with 1.

Comment (nirbhayc) [inline sqlite/ext/comdb2/threadpools.c]: You're right. I will update that.

File: sqlite/ext/comdb2/threadpools.c [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+    systbl_threadpools_cursor *pCur = (systbl_threadpools_cursor *)cur;
+    pCur->pool = thdpool_next_pool(pCur->pool);
+    pCur->rowid++;
+    return SQLITE_OK;
+}
+
+static int systblThreadPoolsEof(sqlite3_vtab_cursor *cur)
+{
+    systbl_threadpools_cursor *pCur = (systbl_threadpools_cursor *)cur;
+    return (pCur->rowid >= listc_size(&threadpools)) ? 1 : 0;
```
Comment (riverszhang89) [inline sqlite/ext/comdb2/threadpools.c]: You only need to `return (pCur->pool == NULL)` here as `thdpool_next_pool()` returns `NULL` if no next node.

**Comment by riverszhang89** [comment]
We adhere to SQLite coding standards (2-space indentation, Hungarian notation and etc) when modifying the module. It is probably not required for our system tables, but is always a good practice.

**Pull request merged**